packages/api-media-karma/src/index.ts

```diff
@@ -81,7 +83,7 @@ export class KarmaMediaAdapter implements MediaAdapter {
   }
 
   async deleteImage(id: string): Promise<boolean> {
-    const response = await fetch(`${this.url}${id}`, {
+    const response = await fetch(`${this.internalURL}${id}`, {
```

Comment:
what happens here if `internalURL` is undefined?

Reply:
It has a default value set in the constructor.

Reply:
but on line 36 you use `this.internalURL ?? this.url`. Both it okay but you should stay consistent.